### ISSUE TITLE
Search over multiple indices

### ISSUE THREAD

**Issue by mengqing** [Zatvobor/tirexs#218]
Is it possible to search over multiple indices?

**Comment by aphistic** [comment]
Yeah, if you use a wildcard '*' in the index name it will search all indices matching that, such as "logstash-*".
